### PR TITLE
removed duplicate parts that are in the readthedocs

### PR DESCRIPTION
**Pull request by shimwell**
## Proposed changes

Along the same lines as #807 this PR is another option for removing the maintainace of the README.md file but simply removing all the duplication. What do you think @RemDelaporteMathurin 

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [x] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Pep8 applied
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,31 +11,17 @@
...
-started to migrate the code base. Shortly after this migration the project
-became open-source and has flourished ever since. The project has grown largely
-due to two contributors in particular (John Billingsley and
-Remi Delaporte-Mathurin) and others have also helped, you can see all those who
-have helped the development in the 
-[Authors.md](https://github.com/ukaea/paramak/blob/main/AUTHORS.md) and in the 
-[GitHub contributions](https://github.com/ukaea/paramak/graphs/contributors).
-The code has been professionally reviewed by [PullRequest.com](https://www.pullrequest.com/) who
-produced a [report](https://github.com/ukaea/paramak/files/5704872/PULLREQUEST.Paramak.Project.Review.pdf) and inline [suggestions](https://github.com/ukaea/paramak/pull/639).
+:point_right: [Installation](https://paramak.readthedocs.io/en/main/#system-installation)
```
Comment (RemDelaporteMathurin) [inline README.md]: perhaps this should go before the publication? or put the publication in the Citing section?

**Review by RemDelaporteMathurin — APPROVED** [review]
That's a great reduction indeed!

Perhaps having one code section "Getting started" would be nice. Like a simple code to create a tokamak geometry or something.

File: README.md (hunk 1 above)
Comment (shimwell) [reply README.md]: I can move it before the publication

**Comment by shimwell** [comment]
Shall I add the "Getting started" section to the read the docs or the README.md.

I think that having zero examples in the readme will steer users to the documentation, so perhaps I can add a getting started section in the documentation

**Pull request merged**